.changeset/grumpy-sloths-check.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/core-components': patch
+'@backstage/plugin-catalog': patch
+---
+
+System Diagram was getting clipped when it was wider than the container.
```

Comment:
```suggestion
Fixed sizing of the System diagram when the rendered graph was wider than the container.
```
Sorry for the small nit here, just wanted to make the language sound more like a change than a problem 😅